lib/utils/host/hostusers.go

```diff
@@ -53,9 +54,18 @@ func GroupAdd(groupname string, gid string) (exitCode int, err error) {
 	cmd := exec.Command(groupaddBin, args...)
 	output, err := cmd.CombinedOutput()
 	log.Debugf("%s output: %s", cmd.Path, string(output))
-	if cmd.ProcessState.ExitCode() == GroupExistExit {
+
+	switch cmd.ProcessState.ExitCode() {
```

Comment:
```suggestion
	switch code := cmd.ProcessState.ExitCode(); code {
```

Then you don't have to call `cmd.ProcessState.ExitCode()` 3 more times.

Reply:
Good call, I totally forgot you could do that in a switch :sweat_smile: